Mathlib/MeasureTheory/Measure/Typeclasses/SFinite.lean

```diff
@@ -538,6 +542,17 @@ instance (priority := 100) IsFiniteMeasure.toSigmaFinite {_m0 : MeasurableSpace
     [IsFiniteMeasure μ] : SigmaFinite μ :=
   ⟨⟨⟨fun _ => univ, fun _ => trivial, fun _ => measure_lt_top μ _, iUnion_const _⟩⟩⟩
 
+/-- A measure on a countable space is sigma-finite iff it gives finite mass to every singleton. -/
+lemma Measure.sigmaFinite_iff_measure_singleton_lt_top [Countable α] :
+    SigmaFinite μ ↔ ∀ a, μ {a} < ∞ where
+  mp _ a := measure_singleton_lt_top
```

Comment:
is there any particular reason for this style (with `where` and `mp` and `mpr` fields)?